run_sets/testsuite.yml

```diff
@@ -16,76 +16,5 @@
 - features/core_srv_users.feature
 - features/core_srv_credentials.feature
 - features/core_srv_create_activationkey.feature
+- features/core_proxy.feature
 - features/core_trad_register_client.feature
-- features/core_min_bootstrap.feature
-- features/core_min_docker_build_image.feature
-- features/core_min_salt_ssh.feature
-- features/core_centos_tradclient.feature
-- features/core_centos_salt.feature
-- features/core_centos_salt_ssh.feature
-# these features sync real channels (last core feature)
-- features/core_srv_sync_channels.feature
-- features/core_srv_setup_wizard.feature
-## CORE feature end ###
-
-# SECONDARY FEATURE:
-
-# **** IDEMPOTENT ******
-- features/min_bootstrap_xmlrpc.feature
-- features/min_ssh_bootstrap_xmlrpc.feature
-- features/trad_need_reboot.feature
-- features/trad_ssh_push.feature
-- features/trad_reboot.feature
-- features/srv_change_password.feature
-- features/min_salt_software_states.feature
-- features/trad_metadata_check.feature
-- features/srv_clone_channel_npn.feature
-- features/trad_cve_id_new_syntax.feature
-- features/trad_weak_deps.feature
-- features/srv_nagios.feature
-- features/srv_cve_audit.feature
-- features/min_salt_install_with_staging.feature
-- features/srv_xmlrpc_activationkey.feature
-- features/allcli_overview_systems_details.feature
```

Comment:
remember to dont  eliminate  this

Reply:
??? why would i do that ?

Reply:
in you pr you are removing this features

Reply:
"oops"

:blush: